package.json

```diff
@@ -3,11 +3,44 @@
   "version": "2.3.0",
   "description": "A git log to release changelog tool",
   "main": "changelog-maker.js",
+  "files": [
```

Comment:
maybe `files` should go above `main`? as `main` should be inside of it. and then `main` will be next to `bin`

Reply:
done